app/controllers/catalog_controller.rb

```diff
@@ -743,6 +743,12 @@ def ot_show
     explorer
   end
 
+  def gtl_buttons
+    # used @gtl_buttons, service catalog list does not need to show view_grid button
+    # other accordions in this explorer show all three buttons
+    @gtl_buttons ||= %w[view_grid view_list view_tile]
+  end
```

Comment:
I don't really understand how is this related and I'm worried that if this is really necessary, then other screens might need it as well. I'd prefer to not have this, will think about it how...

Reply:
@skateman as discussed with you, I will be removing the tree view from this PR, as we really do not have a need for the tree on this screen and we want to implement the toggle of tree view button with a better approach. 
cc @chessbyte @Fryguy @gtanzillo